src/Microsoft.Health.Fhir.Liquid.Converter.FunctionalTests/TestData/Expected/Hl7v2/ADT_A01/ADT01-28-expected.json

```diff
@@ -184,27 +210,24 @@
       }
     },
     {
-      "fullUrl": "urn:uuid:a6091df2-6411-bec2-7f42-36d851d96cbe",
+      "fullUrl": "urn:uuid:",
       "resource": {
-        "resourceType": "Provenance",
-        "id": "a6091df2-6411-bec2-7f42-36d851d96cbe",
-        "agent": [
-          {
-            "type": {
-              "coding": [
-                {
-                  "code": "author",
-                  "system": "http://terminology.hl7.org/CodeSystem/provenance-participant-type"
-                }
-              ]
-            }
-          }
-        ],
-        "recorded": "1988-08-18T11:26:00Z"
+        "resourceType": "Practitioner"
       },
       "request": {
         "method": "PUT",
-        "url": "Provenance/a6091df2-6411-bec2-7f42-36d851d96cbe"
+        "url": "Practitioner/"
+      }
+    },
+    {
```

Comment:
This looks like a hanging resource. It has a resource id, but nothing else. Is a resource getting split somehow?

Reply:
There are circumstances in which we are only able to generate resource ID, and all the other properties can't be generated. Should we delete these resources?